cmd/sponged/handlers/data.go

```diff
@@ -0,0 +1,49 @@
+// Package handlers contains the handler logic for processing requests.
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/ardanlabs/kit/db"
+	"github.com/ardanlabs/kit/web/app"
+	"github.com/coralproject/shelf/internal/sponge/item"
+)
+
+const defaultVersion = 1
```

Comment:
More comments concerning the functionality of the data layer added to the dataHandle element.  Reordered initial declarations and added code separator to better tell the story of the handler.